src/app.js

```diff
@@ -0,0 +1,38 @@
+const myList = (mytodoList) => {
+  const sortedList = mytodoList.list.sort((a, b) => a.index - b.index);
+  const itemsContainer = document.querySelector('.item');
+  let itemsHtml = '';
+  sortedList.forEach((todo) => {
+    itemsHtml += `  <div class="todoItem">
+
+                            <input type="checkbox" class="todoCheck"/>
+                            <input id="${todo.index}" class="itemTodo" type="text" value="${todo.description}" />
+
+                        <button id="${todo.index}" class="delButton"> <i class="fas fa-trash"></i></button>
+                      </div>
+
+    `;
+  });
+  itemsContainer.innerHTML = itemsHtml;
+
+  // delete
+  const removeBtns = document.querySelectorAll('.delButton');
+  removeBtns.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      const element = btn.parentNode;
+      element.remove();
+      mytodoList.deleteList(Number(e.target.parentNode.id));
+      // element.remove();
+    });
```

Comment:
- [ ] I think it would b better to delete all the commented code lines from your code base to make it cleaner and looks professional.

Reply:
This change has been done well